setup.cfg

```diff
@@ -18,21 +18,24 @@ package_dir =
 packages = find:
 python_requires = >=3.6
 install_requires =
-  numpy == 1.21
-  scipy == 1.7
-  matplotlib == 3.5
-  numba == 0.55
-  ipykernel == 6.9   # Allows this conda environment to show up automatically in Jupyter Lab
-  watermark == 2.3    # Generates a summary of package version for use inside Jupyter Notebooks
+    numpy == 1.21
+    scipy == 1.7
+    matplotlib == 3.5
+    numba == 0.55
+    ipykernel == 6.9  # Allows this conda environment to show up automatically in Jupyter Lab
+    watermark == 2.3  # Generates a summary of package version for use inside Jupyter Notebooks
 
-[options.packages.find]
-where = src
+[options.extras_require]
 dev =
-  pytest == 7.1      # Testing
-  pytest-cov == 3.0  # For Coverage testing
-  hypothesis == 6.29  # Property based testing
-  pre-commit == 2.20
-
+    pytest == 7.1       # Testing
+    pytest-cov == 3.0   # For Coverage testing
+    hypothesis == 6.29  # Property based testing
+    pre-commit == 2.20
+    jupyterlab == 3.4.3
 docs =
-  sphinx == 5.0       # For building the documentation
-  myst-nb == 0.16
```

Comment:
these should be under the `extras_require` field, I have placed them there in the PR